docs/developers/index.rst

```diff
@@ -0,0 +1,12 @@
+=========================
+Developer's documentation
```

Comment:
now that i see it in the menu i feel that using "documentation" isn't great as the entire site is devoted to documentation and the whole title is long. the Charm Guide has a section called "Community" with pages:

![image](https://github.com/canonical/microovn/assets/6303808/3aa0d439-427d-4954-bd44-d097c85f9832)

what do you think?

Reply:
I don't have strong opinion on this. `Community` section seem bit vague to me, perhaps `For Contributors`? However consistency across projects is also nice, so I'm not opposed to `Community`. I'll defer to your recommendation.

Reply:
Let's merge this as-is and we can always change it later